src/main/java/bit/Storage.java

```diff
@@ -0,0 +1,170 @@
+package bit;
+
+import java.io.*;
```

Comment:
Try not to import all haha. Only import the ones you need 😀

Reply:
Thanks, I will get rid of the wildcard soon